shopify/utils/utilities.py

```diff
@@ -0,0 +1,64 @@
+from .exceptions import *
```

Comment:
I think * imports are frowned upon. I tend to like the pattern you've got below where you import the module and then access its members:

```python
from . import exceptions

...

    raise exceptions.InvalidIssuerError

...
```